src/controllers/dynakube/activegate/internal/proxy/reconciler_test.go

```diff
@@ -45,7 +45,7 @@ func newTestReconcilerWithInstance(client client.Client) *Reconciler {
 func TestReconcileWithoutProxy(t *testing.T) {
 	t.Run(`reconcile dynakube without proxy`, func(t *testing.T) {
 		r := newTestReconcilerWithInstance(fake.NewClientBuilder().Build())
-		err := r.Reconcile()
+		err := r.Reconcile(context.TODO())
```

Comment:
Nitpick: I think we should start using `context.Background` in tests. So we can use `context.TODO` as an actual TODO. (I might have been for using `context.TODO` in tests, and told people to do this, but I was wrong 😬 )

also I should probably add this in the styleguide or somewhere 😅

Reply:
you don't have to update anything you haven't touched! but if we are touching it right now, lets do it properly

Reply:
agree with @0sewa0 ☝️

Reply:
agree with you :) fixed